cachito/web/app.py

```diff
@@ -110,6 +115,14 @@ def create_app(config_obj=None):
 
     app.register_blueprint(docs)
     app.register_blueprint(api_v1, url_prefix="/api/v1")
```

Comment:
I wonder if this doing anything now that we've removed all of the routes that referenced it?